package.json

```diff
@@ -55,6 +55,7 @@
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-react": "^7.30.0",
     "eslint-plugin-react-hooks": "^4.5.0",
+    "react-test-renderer": "^18.1.0",
     "sass": "^1.52.1",
     "stylelint": "^13.13.1",
     "stylelint-config-standard": "^21.0.0",
```

Comment:
### You have made no changes other than to test files.

![image](https://user-images.githubusercontent.com/27709832/171993411-8d09905e-f8b6-4bac-97f7-362d5547958f.png)

- [ ] **Required:** You are required to follow the [design guideline](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).

- The app has a nav which you have.
- The app has a section containing the world map and the search bar.
- Then the card of cards containing countries and their maps.

Kindly implement them and request another review.

Reply:
@brytebee the API that I am using has no maps, neither does it have continents. I am created my web according the API that I am using.